steps/delius/contact/find-contacts.ts

```diff
@@ -5,8 +5,10 @@ import { Contact } from '../../../test-data/test-data.js'
 
 export async function findContactsByCRN(page: Page, crn: string) {
     await findOffenderByCRN(page, crn)
-    await page.click('#linkNavigation1ContactList')
-    await expect(page).toHaveTitle(/Contact List/)
+    await doUntil(
+        () => page.click('#linkNavigation1ContactList'),
+        () => expect(page).toHaveTitle(/Contact List/)
+    )
```

Comment:
Are you sure you need this retry?  Clicking on the contact list should be pretty stable, is there an issue with it?